sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/MethodShouldNotOnlyReturnConstant.cs

```diff
@@ -69,6 +70,11 @@ protected override void Initialize(SonarAnalysisContext context)
                 SyntaxKind.MethodDeclaration);
         }
 
+        private bool IsVirtual(MethodDeclarationSyntax methodDeclaration)
+        {
+            return methodDeclaration.Modifiers.Any(m => m.IsAnyKind(new SyntaxKind[] { SyntaxKind.VirtualKeyword }));
```

Comment:
`new SyntaxKind[]...` is better to be a `private readonly ISet<SyntaxKind>` or just use `m.IsKind(SyntaxKind.VirtualKeyword)`